### DISCUSSION TITLE
show cpu/gpu frequency and power

### DISCUSSION THREAD

**Discussion by paslavskyit** [seerge/g-helper#688] [Ideas]
for now we only have temps, and fan speed, would be great to see the cpu and gpu frequency and power consumption please if possible

also, i'd help to add ukrainian localization to the app

**Comment by seerge** [comment]
@shprd95 hello,

What is the practical outcome of CPU and especially GPU frequencies (since it's varies per platform / model)? :)

Power consumption (if you mean battery drain) is already there - it appears in battery section when it's discharging. 

And app already has Ukrainian translation (it uses windows language settings). 

Or you can set it manually by adding
``language:"uk",`` to ``%appdata%\ghelper\config.json`` and launching app after.

**Reply by paslavskyit** [reply]
@seerge Hi, i mean cpu/gpu frequencies in mhz to be seen in app and tray on hover
by "power consumption" i mean for gpu (60w or 80w etc) and cpu (15w, 45w etc) and same to be seen in app

**Reply by seerge** [reply]
Monitoring power usage per GPU / CPU is not that easy (as it depends on the drivers / etc), and requires using external libraries such as https://github.com/LibreHardwareMonitor/LibreHardwareMonitor which in turn will eat extra resources / make app more heavy, so I'm not a big fan of adding that :) (i was using this lib in early versions and then removed in order to make app lighter)

Also - each platform Nvidia / AMD has quite nice and convenient overlays that show power draw / etc in games already
